classes/controller/AdminController.php

```diff
@@ -2794,7 +2797,17 @@ public function init()
                 $this->context->cookie->last_activity = time();
             }
         }
-        if ($this->controller_name != 'AdminLogin' && (!isset($this->context->employee) || !$this->context->employee->isLoggedBack())) {
+
+        if (
+            !$this->allowAnonymous
```

Comment:
May be a getter for this attribute could be useful for allow anonymous under conditions.

Reply:
Done

Reply:
:ok_hand: :+1: